libs/verifiable-credentials/src/main/java/uk/gov/di/ipv/core/library/verifiablecredential/helpers/VcHelper.java

```diff
@@ -77,6 +75,28 @@ public static boolean isSuccessfulVcs(List<SignedJWT> vcs) throws ParseException
         return true;
     }
 
+    public static List<VcStoreItem> filterVCBasedOnProfileType(
```

Comment:
Now you've merged Chris's changes, can we commonise this method with his (`CheckExistingIdentityHandler#removeOperationalProfileVcs`)?

They work slightly differently, but are trying to achieve the same thing, and we should be consistent.

Reply:
Done as part of last commit.